### PR TITLE
[FIX] gamification, website_(slides,forum): ensure Marc Demo has karma

### PR DESCRIPTION
**Pull request by flch-odoo**
Without it, like in no-demo testing conditions, tests with rights 
depending on Karma fail.

Task-3575692

### CODE REVIEW

File: addons/gamification/tests/common.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+class HttpCaseWithUserDemoGamification(HttpCaseWithUserDemo):
+    def setUp(self):
+        super().setUp()
+        if not self.user_demo.karma:
+            self.user_demo.karma = 2500
+
+
+class TransactionCaseWithUserDemoGamification(TransactionCaseWithUserDemo):
+    def setUp(self):
```
Comment (thjo-odoo) [inline addons/gamification/tests/common.py:14]: I'll add a reminder for the FW in 16.0 => The setUp was replaced by setUpClass for TransactionCaseWithUserDemo

**Review by thjo-odoo — APPROVED** [review]

**Review by awa-odoo — COMMENTED** [review]
Hello @flch-odoo 

Only one minor comment.

Thank you for this good initiative!

File: addons/gamification/tests/common.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.base.tests.common import HttpCaseWithUserDemo, TransactionCaseWithUserDemo
+
+
+class HttpCaseWithUserDemoGamification(HttpCaseWithUserDemo):
```
[the remark this answers is not in the document]
Comment (flch-odoo) [reply addons/gamification/tests/common.py]: Thank you for the review!

There is a `TestGamificationCommon` in test_challenge.py. I did rename to shorter versions here, for the final touch I guess it should rather be done in master?

**Comment by awa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**